### ISSUE TITLE
.aligned.fa result's file

### ISSUE THREAD

**Issue by madeleinevlt** [tseemann/snippy#283]
Hello, 
I've got a question concerning the .aligned.fa file. In the README, it's written that the aligned.fa file contain a "version of the reference but with - at position with depth=0 and N for 0 < depth < --mincov **(does not have variants)** "

When i took a quikly look in your code, I guess you are using the reference with SNPs applied which come from the .consensus.subs.fa file.

So if I understand well, the  .aligned.fa would contains a version of the reference with SNPs, and not the reference you give to snippy at the beggining.

Is that true ?

**Comment by tseemann** [comment]
You are right. I am using `.conensus.subs.fa`. It seems this has changed between Snippy 3.x and 4.x.  I also patch in `n` for het sites. I will need to update the documentation.  

The `.aligned.fa` file is used internally in conjunction with `.vcf` to build a whole genome alignement. However, in snippy-core it seems I re-call the SNPs and also patch in deletions.

This is somewhat confusing!

**Comment by cizydorczyk** [comment]
So just to clarify,

`.consensus.subs.fa` contains only SNPs implemented on the reference genome (with gaps and hets & Ns masked, does this include complex variants?), `.consensus.fa` is the same but contains ALL variants (including indels), and `.aligned.fa` is JUST gaps and ambiguous bases/hets? And it is the `.vcf` and the `.aligned.fa` files that are used in snippy-core?

**Comment by tseemann** [comment]
My head hurts now!
